Tests/UnitTests/Core/Seeding/HoughTransformTest.cpp

```diff
@@ -0,0 +1,135 @@
+    ActsFatras::Hit::Vector3 tube_pos{tubePos[i][0] * Acts::UnitConstants::mm,
+                                      tubePos[i][1] * Acts::UnitConstants::mm,
+                                      tubePos[i][2] * Acts::UnitConstants::mm};
+    driftCircles.push_back(ActsExamples::DriftCircle(
+        std::move(tube_pos), driftRadius[i], 0.0f, stationInfo[i][0],
+        stationInfo[i][1], stationInfo[i][2], stationInfo[i][3],
+        stationInfo[i][4], stationInfo[i][5]));
+  }
+
+  // configure the binning of the hough plane
+  Acts::HoughTransformUtils::HoughPlaneConfig planeCfg;
+  planeCfg.nBinsX = 1000;
+  planeCfg.nBinsY = 1000;
+
+  // instantiate the peak finder
+  Acts::HoughTransformUtils::PeakFinders::IslandsAroundMaxConfig peakFinderCfg;
+  peakFinderCfg.fractionCutoff = 0.7;
+  peakFinderCfg.threshold = 3.;
+  peakFinderCfg.minSpacingBetweenPeaks = {0., 30.};
+
+  // and map the hough plane to parameter ranges.
+  // The first coordinate is tan(theta), the second is z0 in mm
+  Acts::HoughTransformUtils::HoughAxisRanges axisRanges{-3., 3., -2000., 2000.};
+
+  // create the functions parametrising the hough space lines for drift circles.
+  // Note that there are two solutions for each drift circle and angle
+
+  // left solution
+  auto houghParam_fromDC_left = [](double tanTheta,
+                                   const ActsExamples::DriftCircle& DC) {
+    return DC.y() - tanTheta * DC.z() -
+           DC.rDrift() / std::cos(std::atan(tanTheta));
+  };
+  // right solution
+  auto houghParam_fromDC_right = [](double tanTheta,
+                                    const ActsExamples::DriftCircle& DC) {
+    return DC.y() - tanTheta * DC.z() +
+           DC.rDrift() / std::cos(std::atan(tanTheta));
+  };
+
+  // create the function parametrising the drift radius uncertainty
+  auto houghWidth_fromDC = [](double, const ActsExamples::DriftCircle& DC) {
+    return std::min(DC.rDriftError() * 3.,
+                    1.0);  // scale reported errors up to at least 1mm or 3
+                           // times the reported error as drift circle calib not
+                           // fully reliable at this stage
+  };
+
+  // instantiate the hough plane
+  Acts::HoughTransformUtils::HoughPlane<Acts::GeometryIdentifier::Value>
+      houghPlane(planeCfg);
+
+  // also insantiate the peak finder
+  Acts::HoughTransformUtils::PeakFinders::IslandsAroundMax<
+      Acts::GeometryIdentifier::Value>
+      peakFinder(peakFinderCfg);
+
+  // loop over the true hits
+  for (auto& sh : simHits) {
+    houghPlane.reset();
+
+    for (ActsExamples::DriftCircle& dc : driftCircles) {
+      ActsExamples::muonMdtIdentifierFields idf;
+      idf.multilayer = dc.multilayer();
+      idf.stationEta = dc.stationEta();
+      idf.stationPhi = dc.stationPhi();
+      idf.stationName = dc.stationName();
+      idf.tubeLayer = dc.tubeLayer();
+      idf.tube = dc.tube();
+      auto identifier = compressId(idf);
```

Comment:
for the CI, the identifier could even just be a trivial integer, in case you don't want to worry about identifier construction. The hough utils support arbitrary types from them (in ATLAS, for example, we use pointers to raw data objects as "identifiers")